### ISSUE TITLE
Provide environment variables for all cli flags

### ISSUE THREAD

**Issue by xoxys** [christian-korneck/docker-pushrm#7]
**Describe the problem or bug**
It would be helpful if we could set all configuration options as environemt variables e.g. `FILE_PATH=/myvolume/Readem.md` instead of `--file`.

**docker-pushrm version**
v1.4.3

**Comment by christian-korneck** [comment]
hi @xoxys , thanks for the suggestion. Do you have a particular use case in mind where this would be useful? (Could you give an example?)

Even if you use this tool i.e. with CI tools (like github actions, etc), where env vars play an important role, you can do things like:

```
docker pushrm -f $INPUT_FILE
```

Is there any use case where this doesn’t work?

**Comment by xoxys** [comment]
Hi, you are right I could workaround it and get it to work within my CI but it's currently more complicated as it could be. I'm using Drone CI (and your official docker image) and the current working config looks like this:

```
steps:
- name: readme
  image: chko/docker-pushrm:1
  entrypoint:
  - /docker-pushrm
  command:
  - --file
  - README.md
  - --short
  - "Custom Alpine base image"
  - xoxys/${DRONE_REPO_NAME}
  environment:
    DOCKER_PASS:
      from_secret: docker_password
    DOCKER_USER:
      from_secret: docker_username
```

and that's how it will look like if I would be able to use env vars for every parameter:
```
steps:
- name: readme
  image: chko/docker-pushrm:1
  environment:
    DOCKER_PASS:
      from_secret: docker_password
    DOCKER_USER:
      from_secret: docker_username
    INPUT_FILE: README.md
    REPO_NAME: xoxys/${DRONE_REPO_NAME}
    SHORT_DESCRIPTION: "Custom Alpine base image"
```

Passing env vars from the `environment:` section to the `command:` section will not work because there is no shell in the docker image as you are using `scratch` and it could not pick up env vars on the entrypoint command.

**Comment by xoxys** [comment]
Would you accept a PR or are you already working on an implementation?

**Comment by christian-korneck** [comment]
sure, feel free to submit a PR. Thanks!

**Comment by christian-korneck** [comment]
merged and released in [v1.5.0](https://github.com/christian-korneck/docker-pushrm/releases/v1.5.0). thanks again.

**Comment by xoxys** [comment]
Thank you too for the fast release :+1:
